bindings/python/src/converters.cpp

```diff
@@ -109,6 +109,41 @@ struct tuple_to_pair
     }
 };
 
+template<class T1, class T2>
+struct dict_to_map
```

Comment:
yes, for rename: https://github.com/arvidn/libtorrent/pull/1541/files#diff-9a8153ff01a5fbbc984a9b19dcb49e22R313

That was the storry where I wonder why the getter is not working after I fugured out that it was just a method call.

I have to say my work with c++ is very unproductive, but at least something got accomplished :)